internal/rootcoord/import_manager.go

```diff
@@ -729,3 +729,18 @@ func taskExpired(ti *datapb.ImportTaskInfo) bool {
 		ti.GetState().GetStateCode() != commonpb.ImportState_ImportCompleted &&
 		Params.RootCoordCfg.ImportTaskExpiration <= float64(time.Now().Unix()-ti.GetCreateTs())
 }
+
+func (m *importManager) GetImportFailedSegmentIDs() ([]int64, error) {
```

Comment:
comment

Reply:
what does this mean?

Reply:
add comment for this method. But it seems that you commented in the interface so nvm.

Reply:
ok